docs/website/docs/intro.md

```diff
@@ -17,6 +17,10 @@ from various and often messy data sources into well-structured, live datasets. T
 ```sh
 pip install dlt
 ```
+:::tip
+We recommend using a clean virtual environment for your experiments! Here are [detailed instructions](reference/installation).
```

Comment:
The link does not work in the preview. (https://deploy-preview-1625--dlt-hub-docs.netlify.app/docs/intro/) should it be `../reference/installation`?

```suggestion
We recommend using a clean virtual environment for your experiments! Here are [detailed instructions](../reference/installation).
```